### ISSUE TITLE
Feature Request: Configurable Column Name Support in User Model

### ISSUE THREAD

**Issue by uguranium** [laravel/pulse#312]
Hello,

We've encountered an issue in our application due to a discrepancy in the database schema of the users table. Specifically, our users table does not include a name column, which is causing SQL errors in certain parts of the application.

Issue Encountered:
The application attempts to select a name column from the users table, but since our schema doesn't include this column, it results in the following SQL error:

 ```
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'name' in 'field list' (SQL: select `id`, `name`, `email` from `users` where `users`.`id` in (151, 116, 818162, 6) and `users`.`deleted_at` is null)
 ```
 
 
This issue arises from the following method:
 ```
public function resolveUsers(Collection $ids): Collection {
    // ... existing code ...

    elseif (class_exists(\App\Models\User::class)) {
        return \App\Models\User::whereKey($ids)->get(['id', 'name', 'email']);
    }
    elseif (class_exists(\App\User::class)) {
        return \App\User::whereKey($ids)->get(['id', 'name', 'email']);
    }

    // ... existing code ...
}
 ```
 
Proposed Solution:
We suggest introducing the ability to specify a custom column name to represent the user's name in the configuration. This flexibility would allow the application to accommodate different database schemas more easily, especially in cases where the users table schema deviates from the expected norm (like in our case).

For example, a configuration option in config/pulse.php (or a relevant config file) that allows the developer to specify the actual column to be used as the user's name:
 ```
'user_name_column' => 'firstname',  
 ```
This approach would provide a more dynamic and adaptable solution, allowing for seamless integration with various database designs.

Thank you for considering this feature request. We believe it would enhance the flexibility and robustness of the application.

**Comment by driesvints** [comment]
For this, it's best that you resolve the user: https://laravel.com/docs/10.x/pulse#dashboard-resolving-users

**Comment by uguranium** [comment]
I encountered an issue with the provided example: the Pulse::user method was resulting in a "method not found" error. To resolve this, I switched to using Pulse::users() instead of Pulse::user(). This change successfully eliminated the error.

However, while the error is no longer present, I've noticed that some columns are still not being displayed as expected. Despite this, the immediate problem has been addressed. Thank you for your assistance.

https://laravel.com/docs/10.x/pulse#dashboard-resolving-users

```
use Laravel\Pulse\Facades\Pulse;
Pulse::user(fn ($user) => [
        'name' => $user->name,
        'extra' => $user->email,
        'avatar' => $user->avatar_url,
]);
```
